test/celeritas/random/XorwowRngEngine.test.cc

```diff
@@ -205,6 +205,62 @@ TEST_F(XorwowRngEngineTest, moments)
     tally.check(num_samples * num_seeds, 1e-3);
 }
 
+TEST_F(XorwowRngEngineTest, jump)
```

Comment:
It would be also good to show that discarding `N` produces the same value as evaluating `N` times? (For small N obviously; there's no need to discard 2^67...)

Reply:
I show this for all `N` between 0 and 2^16, I think that's what you meant?

Reply:
Oh. Sorry I missed that!

Reply:
Ok I see, it's being done implicitly through initialization rather than through discard. 🤔

Reply:
Ahh right that makes sense.